io.github.ctlcltd.e2se.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.ctlcltd.e2se
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: e2-sat-editor
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
```

Comment:
if this really support wayland then it should be `fallback-x11`

Reply:
this depends on Qt6 runtime, org.kde.Platform version 6.4 is used. I can also put "fallback-x11" if that's a requirement. In the documentation there is "--socket=fallback-x11 - show windows using X11, **if Wayland is not available**", I don't know if there is Wayland installed in user's device, I would prefer to support both Wayland, both X11

Reply:
>  I don't know if there is Wayland installed in user's device, 

You don't have to.